core/src/trezor/ui/layouts/tt.py

```diff
@@ -407,4 +567,67 @@ def confirm_modify_fee(
     text.br_half()
     text.normal("Transaction fee:")
     text.bold(total_fee_new)
-    return interact(ctx, HoldToConfirm(text), "modify_fee", ButtonRequestType.SignTx)
+    await raise_if_cancelled(
+        interact(ctx, HoldToConfirm(text), "modify_fee", ButtonRequestType.SignTx)
+    )
+
+
+async def confirm_coinjoin(
+    ctx: wire.GenericContext, fee_per_anonymity: Optional[str], total_fee: str
+) -> None:
+    text = Text("Authorize CoinJoin", ui.ICON_RECOVERY, new_lines=False)
+    if fee_per_anonymity is not None:
+        text.normal("Fee per anonymity set:\n")
+        text.bold("{} %\n".format(fee_per_anonymity))
+    text.normal("Maximum total fees:\n")
+    text.bold(total_fee)
+    await raise_if_cancelled(
+        interact(ctx, HoldToConfirm(text), "coinjoin_final", ButtonRequestType.Other)
+    )
+
+
+# TODO cleanup @ redesign
+async def confirm_sign_identity(
+    ctx: wire.GenericContext, proto: str, identity: str, challenge_visual: Optional[str]
+) -> None:
+    lines: List[TextContent] = []
+    if challenge_visual:
+        lines.append(challenge_visual)
+
+    lines.append(ui.MONO)
+    lines.extend(chunks(identity, 18))
+
+    text = Text("Sign %s" % proto)
+    text.normal(*lines)
+    await raise_if_cancelled(
+        interact(ctx, Confirm(text), "sign_identity", ButtonRequestType.Other)
+    )
+
+
+async def confirm_signverify(
+    ctx: wire.GenericContext, coin: str, message: str, address: str = None
```

Comment:
`address: str = None` seems like an invalid type declaration since some changes in mypy. Is this covered by typecheck?

Reply:
It's illegal wrt [PEP484](https://www.python.org/dev/peps/pep-0484/#union-types) but mypy ([still](https://github.com/python/mypy/issues/9091)) allows it. This behavior can be [turned off](https://github.com/trezor/trezor-firmware/blob/ba326865f1b9cacf25eb7d758ba45385cdc41fb7/setup.cfg#L48) but we rely on in in ~300 places. I'll fix the usage in this PR and will look into the rest of the codebase afterwards.

Reply:
also worth considering is switching to the new-(new-)style declarations with `str | None`

Reply:
you can configure mypy to assume python 3.10

Reply:
Fixed in 4b27e3b0d265ae8a0d16afc78b3c3e688e1e9054.